server/@types/approved-premises/index.d.ts

```diff
@@ -134,6 +134,7 @@ declare module 'approved-premises' {
     name: string
     title: string
 
```

Comment:
Just a thought - I think an abstract class would more closely match what this interface is representing. 
It might be too much work for now but I think it would save us from having to make changes like this in lots of different locations

Reply:
Yeah, I think let's leave it like this for now, and see what shakes out